### ISSUE TITLE
Toolbar: Clarify which element should be in focus order on page load

### ISSUE THREAD

**Issue by 0ddfell0w** [w3c/aria-practices#582]
The toolbar entry states ["When the toolbar receives focus, focus is set on the first control that is not disabled. Optionally, if the toolbar has previously contained focus, focus is set on the control that last had focus."](https://github.com/w3c/aria-practices/blob/master/aria-practices.html#L2465)

It's not clear which element should be in the focus order upon page load. Either it's (A) the element with role=toolbar or (B) the first control in the toolbar

(A) seems odd in that the toolbar has an initial state that cannot be recovered to once an arrow key is pressed on it.
(B) seems more natural, but should be spelled out more explicitly in the spec

**Comment by mcking65** [comment]
@0ddfell0w, are you interpreting the phrase "toolbar receives focus" as the toolbar element is focused? The toolbar itself is not focusable. Would it be more clear if it was phrased:

>When focus enters the toolbar, focus is set on the first control that is not disabled. Optionally, if the toolbar has previously contained focus, focus is set on the control that last had focus.

The APG generally does not mention page load. That is not the only time that a UI may be initialized. But, in this case, the behavior of focusing the first item may be every time focus enters the toolbar. Or, it may be just the first time after initialization. That is a design decision.

**Comment by mcking65** [comment]
@0ddfell0w, This issue is resolved with commit 66b5128. Please re-open if this solution is not sufficient.
